olive/passes/onnx/conversion.py

```diff
@@ -24,6 +24,17 @@
 logger = logging.getLogger(__name__)
 
 
+# Note: This should move to OliveModel itself
+def get_io_config(model):
```

Comment:
Can you add a type hint for `model: PyTorchModel` and output?

Reply:
Done!